lib/active_zuora/generator.rb

```diff
@@ -196,8 +196,7 @@ def add_extra_customizations
       customize 'RatePlanCharge' do
         include LazyAttr
         exclude_from_queries :overage_price, :included_units,
-          :discount_amount, :discount_percentage, :rollover_balance, :price
```

Comment:
This doesn't exclude price, it *includes* it. It's actually a behavioral reversal from the main branch. Our VCR cassettes were recorded on an old version of the gem before it started excluding price from the service query. The _correct_ fix here is actually to re-record the cassette but I talked with @darbyfrey about that yesterday, and it seems like a giant pain in the ass.

So, effectively, this is us being lazy and dumb. Which is why it's in a fork. Perhaps I should add a comment here?